dev/src/lobster/parser.h

```diff
@@ -525,6 +525,11 @@ struct Parser {
             case T_STRTYPE:   dest = type_string;     lex.Next(); break;
             case T_COROUTINE: dest = type_coroutine;  lex.Next(); break;
             case T_RESOURCE:  dest = type_resource;   lex.Next(); break;
+            case T_LAZYEXP: {
+                lex.Next();
+                funarg->flags = ArgFlags(funarg->flags | AF_EXPFUNVAL);
```

Comment:
This should probably error if `funarg` is null, e.g. `struct A { x:lazy_expression }`.. otherwise LGTM!

Reply:
I've added the error message.